resources/js/components/Admin/Box.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-    <div class="box-wrapper" @mousedown="function(e) { selectBox(e) }" @mouseup="dragEnd()">
+    <div class="box-wrapper" @mousedown="function(e) { selectBox(e) }" @mouseup="mouseUp()">
 
-        <div :class="selected ? 'box selected-box' : 'box'"
-             :style="{top: this.geom[0] + 'px', left: this.geom[1] + 'px', width: this.geom[2] + 'px', height: this.geom[3] + 'px'}">
+        <div :class="selected ? 'box selected-box' : 'box'
+"             :style="{top: this.geom[0] + 'px', left: this.geom[1] + 'px', width: this.geom[2] + 'px', height: this.geom[3] + 'px'}">
```

Comment:
I think this double-quote should stay in the same row with class. If you were familiar with eslint, I recommend pulling new code from master and let's eslint do its job 😄.

Reply:
Thanks! I will take a look at eslint.